prepare-module.sh

```diff
@@ -10,14 +10,18 @@ set -u
 set -o pipefail
 
 mkdir -p lambda-files
+echo "* Compiling TypeScript sources"
 npm run-script build
 cp package.json lambda-files
 cd lambda-files || exit "Directory 'lambda-files' does not exist"
+echo "* Installing Javascript dependencies for Lambda"
 npm install --no-package-lock --production
```

Comment:
isn't installing without package lock dangerous? can this lead into situation where we have different dependency versions when testing locally and when deploying to production? perhaps we can avoid this situation by copying the package lock over as well.

Reply:
Yes, that is true. Fixed.